packages/cli-kit/src/public/node/themes/api.ts

```diff
@@ -51,7 +51,7 @@ export async function deleteThemeAsset(id: number, key: Key, session: AdminSessi
   const response = await request('DELETE', `/themes/${id}/assets`, session, undefined, {
     'asset[key]': key,
   })
-  return Boolean(response.json.message)
+  return response.status === 200
```

Comment:
Let's add a comment explaining the API behavior on top of this?

Reply:
Personally, I believe we don't need the comment because the unit tests already document the most intriguing scenarios (which involve the two kinds of 200 responses).

I will proceed with the merge to unblock the release, but please feel free to add an extra test case to the 403 scenario and/or include the comments in a following PR, @jamesmengo :)